Sources/ZipkinRecordingStrategy/SpanKind+Encodable.swift

```diff
@@ -0,0 +1,32 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Jaeger Client Swift open source project
+//
+// Copyright (c) 2020 Moritz Lang and the Jaeger Client Swift project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Tracing
+
+extension SpanKind: Encodable {
+    public func encode(to encoder: Encoder) throws {
+        var container = encoder.singleValueContainer()
+        switch self {
+        case .client:
+            try container.encode("CLIENT")
+        case .server:
+            try container.encode("SERVER")
+        case .consumer:
+            try container.encode("CONSUMER")
+        case .producer:
+            try container.encode("PRODUCER")
+        case .internal:
+            try container.encode("SERVER")
+        }
+    }
+}
```

Comment:
those match zipkin well right?

Reply:
Yes, not sure what to default to for `.internal` though. The Java client returns `null` for anything except `.client`, `.server`, `.consumer`, and `.producer`: https://github.com/jaegertracing/jaeger-client-java/blob/dfb9d8b9dc849f2e6f880b44b26fb77dfc894a02/jaeger-zipkin/src/main/java/io/jaegertracing/zipkin/internal/V2SpanConverter.java#L64

Reply:
I decided to follow the Java client in encoding `null` as Zipkin's docs also declare `span.kind` as `nullable`: https://zipkin.io/zipkin/2.11.0/zipkin/zipkin2/Span.html#kind--